build/images/Dockerfile.octant.ubuntu

```diff
@@ -0,0 +1,27 @@
+FROM golang:1.12 as antrea-build
+
+COPY . /antrea
+
+WORKDIR /antrea
+
+RUN make bin
+
+
+FROM ubuntu:18.04
+
+LABEL maintainer="Antrea <projectantrea-dev@googlegroups.com>"
+LABEL description="A docker image to deploy octant and antrea-octant-plugin."
+
+USER root
+
+# Get and install octant v0.8.0
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends wget && \
+    wget -q https://github.com/vmware-tanzu/octant/releases/download/v0.8.0/octant_0.8.0_Linux-64bit.deb --no-check-certificate && \
+    dpkg -i octant_0.8.0_Linux-64bit.deb && \
+    rm -rf octant_0.8.0_Linux-64bit.deb /var/cache/apt/* /var/lib/apt/lists/* && \
+    apt-get remove -y wget
```

Comment:
Something tells me that it could be better to do the `apt-get remove -y wget` (or `apt-get remove -y wget ca-certificates` after you add the extra package) *before* removing `/var/cache/apt/*` and `/var/lib/apt/lists/*`.